### PR TITLE
Apply page

### PR DESCRIPTION
**Pull request by chriswongtv**

### CODE REVIEW

**Comment by Jeremyyang920** [comment]
Can we indicate which questions on the application are mandatory.

**Comment by chriswongtv** [comment]
@Jeremyyang920 done

File: app/client/views/application/application.html [hunk 1]
```diff
@@ -1,76 +1,154 @@
...
+              name="major"
+              ng-model="user.profile.major">
+              <option value="">Major</option>
+              <option value="CS">Computer Science</option>
+              <option value="CGS">Computer Game Science</option>
+              <option value="CSE">Computer Science and Engineering</option>
+              <option value="INF">Informatics</option>
+              <option value="SE">Software Engineering</option>
+              <option value="BIM">Business Information Management</option>
+              <option value="others">Others</option>
```
Comment (madhavgharmalkar) [inline app/client/views/application/application.html]: Change this to other

**Comment by Jeremyyang920** [comment]
Can you use this page here
https://www.admissions.uci.edu/study/programs.php
and select engineering, physical sciences and ICS and update our current list of majors. I want to make sure to have the engineering fields in there or like math and physics majors.

**Comment by chriswongtv** [comment]
@Jeremyyang920 done

File: app/client/views/application/application.html [hunk 2]
```diff
@@ -1,76 +1,171 @@
...
+              <option value="ENG">Engineering</option>
+              <option value="ENE">Environmental Engineering</option>
+              <option value="ENS">Environmental Science</option>
+              <option value="INF">Informatics</option>
+              <option value="MSE">Materials Science Engineering</option>
+              <option value="MAT">Mathematics</option>
+              <option value="MCE">Mechanical Engineering</option>
+              <option value="PHY">Physics</option>
+              <option value="SE">Software Engineering</option>
+              <option value="others">Others</option>
```
Comment (madhavgharmalkar) [inline app/client/views/application/application.html]: Change 'Others' to 'Other'

**Review by madhavgharmalkar — CHANGES_REQUESTED** [review]

File: app/client/views/application/application.html (hunk 2 above)
Comment (chriswongtv) [reply app/client/views/application/application.html]: @madhavgharmalkar done

**Review by madhavgharmalkar — APPROVED** [review]

**Pull request merged**